### PR TITLE
Update Get-CsOnlineUser.md

### PR DESCRIPTION
**Pull request by isrumnon**

### CODE REVIEW

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by yogkumgit** [comment]
@get-itips @dariomws @tseward

File: skype/skype-ps/skype/Get-CsOnlineUser.md [hunk 1]
```diff
@@ -5,8 +5,8 @@ applicable: Microsoft Teams, Skype for Business Online
 title: Get-CsOnlineUser
 schema: 2.0.0
 manager: bulenteg
-author: tomkau
-ms.author: tomkau
+author: isrumnon
+ms.author: isrumnon@microsoft.com
```
Comment (dariomws) [inline skype/skype-ps/skype/Get-CsOnlineUser.md:9]:
```suggestion
ms.author: isrumnon
```

You need to add your MS alias here, not the entire email address.

**Review by dariomws — CHANGES_REQUESTED** [review]

**Comment by yogkumgit** [comment]
@isrumnon

**Review by dariomws — APPROVED** [review]

**Comment by yogkumgit** [comment]
@get-itips @tseward

**Review by get-itips — APPROVED** [review]

**Review by tseward — APPROVED** [review]

**Comment by yogkumgit** [comment]
@tiburd Please merge this PR. Thanks

**Pull request merged**